blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/formula_5_14.py

```diff
@@ -26,6 +26,7 @@ def __init__(
         ----------
         l_0 : M
             [:math:`l_{0}`] Effective length [:math:`m`].
+            Use your own implementation of this value or use :class: `Form5Dot15EffectiveLengthBraced`.
```

Comment:
add 'integration' test to check this.

Reply:
Do you have a example of this? I see this sentence a lot, but dont see any integration tests.
And what is the point? In the end, it is just a value plugged into a formula.